sway/container.c

```diff
@@ -340,9 +340,16 @@ swayc_t *new_floating_view(wlc_handle handle) {
 	struct wlc_geometry geometry;
 	wlc_view_get_visible_geometry(handle, &geometry);
 
-	// give it requested geometry, but place in center
-	view->x = (swayc_active_workspace()->width - geometry.size.w) / 2;
-	view->y = (swayc_active_workspace()->height- geometry.size.h) / 2;
+	// give it requested geometry, but place in center if possible
+	// in top left otherwise
+	if (geometry.size.w != 0)
```

Comment:
Add braces to `if` and `else` please, even though they only contain one statement.